### PR TITLE
resize: Enable resource type/name syntax

### PR DESCRIPTION
**Pull request by 0xmichalis**
@smarterclayton fyi

### CODE REVIEW

File: pkg/kubectl/cmd/resize.go [hunk 1]
```diff
@@ -65,7 +65,7 @@ func NewCmdResize(f *cmdutil.Factory, out io.Writer) *cobra.Command {
 
 func RunResize(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command, args []string) error {
 	count := cmdutil.GetFlagInt(cmd, "replicas")
-	if len(args) != 2 || count < 0 {
+	if count < 0 {
```
Comment (smarterclayton) [inline pkg/kubectl/cmd/resize.go]: Need to either handle multiple infos or return an error if there is more than one (down in r.Infos()

Comment (kargakis) [inline pkg/kubectl/cmd/resize.go]: done

File: pkg/kubectl/cmd/resize.go [hunk 2]
```diff
@@ -94,6 +94,9 @@ func RunResize(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command, args []str
 	if err != nil {
 		return err
 	}
+	if len(infos) > 1 {
+		return fmt.Errorf("Multiple resources provided: %v", args)
```
Comment (smarterclayton) [inline pkg/kubectl/cmd/resize.go]: Probably needs to match other commands that don't allow multiple.  Also should be in standard form (lowercase first letter)

Comment (kargakis) [inline pkg/kubectl/cmd/resize.go]: Agreed. I will make those changes in this PR.

Comment (kargakis) [inline pkg/kubectl/cmd/resize.go]: I've edited `describe` too. Other commands either support multiple resources or are supproting just one specific resource (so they need different argument handling).

**Comment by smarterclayton** [comment]
LGTM

**Comment by smarterclayton** [comment]
ok to test

**Pull request merged**